pkg/lib/machine-dialogs.js

```diff
@@ -74,7 +75,8 @@
         "invalid-hostkey" : translate_and_init(invalid_hostkey_tmpl),
         "not-supported" : translate_and_init(not_supported_tmpl),
         "sync-users" : translate_and_init(sync_users_tmpl),
-        "unknown-hostkey" : translate_and_init(unknown_hosts_tmpl)
+        "unknown-hostkey" : translate_and_init(unknown_hosts_tmpl),
+        "unknown-host" : translate_and_init(unknown_hosts_tmpl)
```

Comment:
OOI, what is the semantic difference between these two? Is "unknown-hostkey" just for the case that the destination name/IP is already known, but the key changed? Because the latter would be an entirely different "class" of error which should never be overridable in our UI, but we should obviously handle an entirely unknown host. I thought that "changed key" case was "invalid-hostkey".

The code below treats these two pretty much equivalently, so I wonder if the two error codes could just be unified?